src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/single_op_graph.cpp

```diff
@@ -14,6 +14,258 @@ std::shared_ptr<ov::Model> generate(const std::shared_ptr<ov::op::Op> &node) {
     return nullptr;
 }
 
+std::shared_ptr<ov::Model> generate(const std::shared_ptr<ov::op::v8::AdaptiveAvgPool> &node) {
+    const auto data = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, Shape{1, 6, 8, 9});
+    const auto out_shape = ov::op::v0::Constant::create<int32_t>(element::i64, Shape{2}, {5, 7});
```

Comment:
Any reason to use static shape for parameter? If not, please use partial shape or makeDynamicParams for Dynamic shape feature.

Reply:
Noted. They'll be fixed.